Toggl.iOS/ViewControllers/ReportsViewController.KeyCommands.cs

```diff
@@ -27,6 +37,15 @@ protected override void ConfigureKeyCommands()
             modifierFlags: UIKeyModifierFlags.Command,
             propertyList: null);
 
+        private UIKeyCommand shortcutCommand(string input, string title)
+            => UIKeyCommand.Create(
+            title: title,
```

Comment:
If `DateRangePeriod` is sent in as an argument instead of a string `title`, just use
```suggestion
            title: period.ToHumanReadableString(),
```